format_ideas/gaussian1d-option-3.yaml

```diff
@@ -0,0 +1,56 @@
+# Option 3: No parameter/input/output tags.  Are those even useful?
+
+
+!transform/tags/gaussian1d-1.0
+  name: Minimal single model
+
+  parameters:
+    amplitude: 1.0
+    mean: 2.0
+    stddev: 3.0
+
+  inputs: [x]
+
+  outputs: [y]
+
+
+!transform/tags/gaussian1d-1.0
+  name: Single model with units and bounds and constraints
+
+  parameters:
+    amplitude:
+      value: 1.0
+      unit: !core/tags/unit-1.0 m
+      bounds: [0.0, 2.0]
+
+    mean:
+      value: 2.0
+      unit: !core/tags/unit-1.0 s
+      fixed: true
+
+    stddev:
+      value: 3.0
+      unit: !core/tags/unit-1.0 s
+      bounds: [2.0, 4.0]
+
+  inputs: !!omap
+    x:
+      unit: !core/tags/unit-1.0 s
+      bounds: [-10, 10]
+
+  outputs: !!omap
+    y:
+      unit: !core/tags/unit-1.0 m
+
+
+!transform/tags/gaussian1d-1.0
+  name: Minimal model set
+
+  parameters:
+    amplitude: [1.0, 1.1, 1.2, 1.3, 1.4, 1.5]
```

Comment:
@perrygreenfield do you think we need to allow ndarray here?  I mean, allow the model set parameter values to be stored in a binary block?  Or is it better to leave them as list which forces them to always be inline?

Reply:
That nonlinear pixel response model set would have millions of values, so yes, ndarray (and ndarrays have been used for polynomials already, @nden can clarify.

Reply:
Whoa!  Very well, ndarray it is!